packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.js

```diff
@@ -266,4 +293,8 @@ export default class QuanticSort extends LightningElement {
       ({label, value, criterion}) => ({label, value, criterion})
     );
   }
+
+  get label() {
+    return this.variants[this.variant].label;
```

Comment:
To avoid crashing if someone passes a wrong variant.

This could've triggered a `cannot read property 'label' of undefined.`

```suggestion
    return this.variants[this.variant]?.label;
```